pkgs/development/libraries/quickfix/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool }:
+
+let
+  name = "quickfix";
```

Comment:
```suggestion
  pname = "quickfix";
```

Reply:
These could also just be directly defined in the `mkDerivation` block

Reply:
Acted on your suggestion to use a direct definition, and dropped the `let` bindings altogether.